### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by wesrice**
This PR adds tests concerning the parsing of api spec files.

### CODE REVIEW

File: src/utils/openapi.ts [hunk 1]
```diff
@@ -28,3 +28,21 @@ export const isArraySchemaObject = (
 export const isNonArraySchemaObject = (
   obj: OpenAPIV3.SchemaObject
 ): obj is OpenAPIV3.NonArraySchemaObject => !isArraySchemaObject(obj);
+
+
+// export type Property = {
```
Comment (kevcenteno) [inline src/utils/openapi.ts:33]: Can these be removed?

File: package.json [hunk 2]
```diff
@@ -68,6 +68,7 @@
     "@typescript-eslint/eslint-plugin": "^4.4.0",
     "@typescript-eslint/parser": "^4.4.0",
     "babel-eslint": "^10.1.0",
+    "babel-jest": "^26.5.2",
```
Comment (kevcenteno) [inline package.json:71]: Was this a peer dependency?

**Review by kevcenteno — CHANGES_REQUESTED** [review]
This is way clearer now with tests.  Thank you!  Just a couple of comments

File: package.json (hunk 2 above)
Comment (wesrice) [reply package.json:71]: If I recall correctly, yes, this was added to mitigate one of those peer dependency Yarn warnings.

File: src/utils/openapi.ts (hunk 1 above)
Comment (wesrice) [reply src/utils/openapi.ts]: These have been removed.

**Pull request merged**